### ISSUE TITLE
[css-conditional-5] Add inline/block/x/y values to `overflowing` scroll-state query?

### ISSUE THREAD

**Issue by fantasai** [w3c/csswg-drafts#11183]
The `overflowing` scroll-state query has the values `none | top | right | bottom | left | block-start | inline-start | block-end | inline-end `, which indicate each of the sides of the box. But sometimes people want to know about an axis, not one side of it specifically.

Would it make sense to add `x`/`y/`inline`/`block` values to this query as well? They would be true if there is scrollable overflow in either direction along the named axis.

**Comment by tabatkins** [comment]
Yeah, I think this makes sense as a shorthand for having to write `overflowing(left) or overflowing(right)`

**Comment by yisibl** [comment]
Yes, that makes perfect sense. I've found in my practice that many times I want to know if there are horizontal or vertical scrollbars.

**Comment by lilles** [comment]
Proposed async resolution: add inline/block/x/y values to scrollable scroll-state() query

**Comment by astearns** [comment]
The CSSWG will automatically accept this resolution one week from now if no objections are raised here. Anyone can add an emoji to this comment to express support. If you do not support this resolution, please add a new comment.

Proposed Resolution: add inline/block/x/y values to scrollable scroll-state() query

**Comment by astearns** [comment]
RESOLVED: add inline/block/x/y values to scrollable scroll-state() query

**Comment by lilles** [comment]
Test coverage:

http://wpt.live/css/css-conditional/container-queries/scroll-state/at-container-scrollable-parsing.html
http://wpt.live/css/css-conditional/container-queries/scroll-state/scroll-state-scrollable-axis.html
